contracts/exchange/test/balance_stores/local_balance_store.ts

```diff
@@ -1,28 +1,42 @@
+import { constants } from '@0x/contracts-test-utils';
 import { assetDataUtils } from '@0x/order-utils';
 import { AssetProxyId } from '@0x/types';
 import { BigNumber } from '@0x/utils';
 import * as _ from 'lodash';
 
 import { BalanceStore } from './balance_store';
+import { TokenContractsByName, TokenOwnersByName } from './types';
 
 export class LocalBalanceStore extends BalanceStore {
```

Comment:
Optional: We will need to add support for `eth` transfers so that we can test `eth` protocol fees. I think that could fit into this PR.